src/main/java/uk/gov/ons/ctp/response/casesvc/message/notification/CaseNotification.java

```diff
@@ -41,10 +41,11 @@
 
   @XmlElement(required = true)
   protected String caseId;
-
-  @XmlElement(required = true)
+  @XmlElement(nillable = true)
```

Comment:
Add a space above this to make it easier to read

Reply:
👍